src/Symfony/Component/Routing/Loader/AnnotationDirectoryLoader.php

```diff
@@ -18,9 +18,11 @@
  * AnnotationDirectoryLoader loads routing information from annotations set
  * on PHP classes and methods.
  *
+ * @deprecated since Symfony 6.4, use  {@see AttributeDirectoryLoader} instead
+ *
```

Comment:
If this class is deprecated, then there should be a trigger_deprecation here at the top of it.

Reply:
I agree but this triggers deprecations in a lot of places, like FWB or Security bundle functional tests. There are place like `src/Symfony/Component/DependencyInjection/ContainerAwareInterface.php` where we did not put the `trigger_deprecation` call. Maybe we should do the same here? :thinking: Because even if it's deprecated, we still need to extends these classes with the current strategy

Reply:
I managed to add it :slightly_smiling_face: